mandatory/2-function-creation.js

```diff
@@ -15,26 +23,63 @@ function isAcceptableUser(userAge, isLoggedIn) {}
     is applieds and 142.5 should be returned)
 */
 
-function applyDiscount(totalPrice) {}
+function applyDiscount(totalPrice) {
+  let fullDiscount = (totalPrice * 10)/100;
+  let smallDiscount = (totalPrice * 5)/100;
+  if (totalPrice >= 200){
+  return newPrice = totalPrice - fullDiscount;
+  } else {
+  return newPrice = totalPrice - smallDiscount;
+  }
+}
 
 /*
   Complete the function to print to the console the odd numbers between 1 and limit (use a while loop):
-  */
-function printOddNumbers(limit) {}
+*/
+
+
+
+function printOddNumbers(limit) {
+  let lowNumber = 1;
+  while (lowNumber < limit) {
+    console.log(lowNumber);
+  lowNumber = lowNumber + 2;
+  }
+}
+
 
 /*
   Complete the buyTwoGetTheCheapestFree function: if user buys two items, the cheapest item will be free!
   The function should return the price to be paid once the discount is applied.
 */
-function buyTwoGetTheCheapestFree(price1, price2) {}
+
+function buyTwoGetTheCheapestFree(price1, price2) {
+  if (price1 && price2) {
```

Comment:
Looks perfect! I think the code will run without line 57?

Reply:
Yes you are right thank you for pointing it out! 
In this case we do not need to check if the user is buying both items because it's implied in the price comparison (you cannot compare if you only have 1 item!)